### ISSUE TITLE
add connectionProvider option to createServer

### ISSUE THREAD

**Issue by rom1504** [PrismarineJS/node-minecraft-protocol#829]
to support non tcp use cases

**Comment by redbrain** [comment]
WebRTC should be a possible setting for this option, as it's the standard for inter-browser communication, a likely non-TCP use case.

**Comment by rom1504** [comment]
Yes this could be part of an example (using https://github.com/feross/simple-peer )
However nmp itself should not depend on webrtc and instead simply expose a connectionProvider option
